src/services/postService.js

```diff
@@ -1,12 +1,61 @@
+const uuid = require("uuid");
 const { throwIfError } = require('../utilities/dynamoUtilities');
-const { sendPost } = require("../repository/postDAO");
+const postDAO = require("../repository/postDAO");
 
-async function createPost(username, text, score, title){
-    const post = await sendPost(username, text, score, title);
-    throwIfError(post);
-    return {message: "Post created successfully"};
+
+async function createPost(username, description, score, title){
+    const post = {
+        class: "post",
+        itemID: uuid.v4(),
+        postedBy: username,
+        description,
+        score,
+        title,
+        isFlagged: 0
+    }
+    const result = await postDAO.sendPost(post);
+    throwIfError(result);
+    delete (post.class);
+    return post;
+}
+
+async function updatePost(id, post, attributes) {
+    Object.keys(attributes).forEach((key) => {
```

Comment:
I like this, it fills in the empty properties automatically, and you don't have to go back and update it later if new properties are added.